views/home.js

```diff
@@ -37,51 +44,94 @@ const views: ViewType[] = [
   {view: 'OlevilleView', title: 'Oleville', icon: 'mouse-pointer', tint: c.grapefruit},
 ]
 
+const Touchable = ({children, onPress}: {onPress: () => any, children?: Element<any>}) => {
+  return Platform.OS === 'ios'
+    ? <TouchableOpacity onPress={onPress} activeOpacity={0.65}>{children}</TouchableOpacity>
+    : <TouchableNativeFeedback onPress={onPress}>{children}</TouchableNativeFeedback>
```

Comment:
@hawkrives thoughts? Any drawbacks from extracting into our own component to represent the two?